src/tikzplotly/_polar.py

```diff
@@ -0,0 +1,198 @@
+from ._axis import Axis
+from ._utils import option_dict_to_str
+from ._tex import tex_addplot
+from ._color import convert_color
+from ._dataContainer import DataContainer
+from warnings import warn
+import numpy as np
+
+def get_polar_coord(trace, axis: Axis, data_container: DataContainer):
+    polar_layout = getattr(axis.layout, 'polar')
+    if polar_layout:
+        # Angular Axis
+        angularaxis = getattr(polar_layout, 'angularaxis')
+        if angularaxis:
+            # Rotation
+            rotation = getattr(angularaxis, 'rotation')
+            if rotation is None:
+                rotation = 0
+            else:
+                axis.add_option("rotate", rotation)
+                axis.add_option("xticklabel style", f"{{anchor=\\tick+{rotation}+180}}")
+                axis.add_option("yticklabel style", f"{{anchor=\\tick-{rotation}-90}}")
+
+            # Direction
+            direction = getattr(angularaxis, "direction", "counterclockwise")
+            if direction == "clockwise":
+                axis.add_option("y dir", "reverse")
+                axis.add_option("xticklabel style", f"{{anchor={rotation}-\\tick+180}}")
+
+            # Period
+            period = getattr(angularaxis, "period")
+
+            # Category
+            angular_categoryorder = getattr(angularaxis, 'categoryorder', 'trace')
+            angular_categoryarray = getattr(angularaxis, 'categoryarray', None)
+
+        # Radial Axis
+        radialaxis = getattr(polar_layout, 'radialaxis')
+        if radialaxis:
+            # Type
+            radial_axis_type = getattr(radialaxis, 'type', None)
+            if radial_axis_type is not None and radial_axis_type not in ['-', 'linear', 'category']:
+                warn(f"Polar: Radial axis type {radial_axis_type} is not supported yet.")
+
+            # Category
+            radial_categoryorder = getattr(radialaxis, 'categoryorder', 'trace')
+            radial_categoryarray = getattr(radialaxis, 'categoryarray', None)
+
+            # Range
+            sector = getattr(radialaxis, 'range')
+            if sector and len(sector) > 1:
+                axis.add_option("ymin", sector[0])
+                axis.add_option("ymax", sector[1])
+
+        # Sector
+        sector = getattr(polar_layout, 'sector')
+        if sector and len(sector) > 1:
+            axis.add_option("xmin", sector[0])
+            axis.add_option("xmax", sector[1])
+
+    theta = [t if t is not None else '' for t in trace.theta]
+    r = [val if val is not None else 'nan' for val in trace.r]
+
+    thetaunit = getattr(trace, "thetaunit", "degrees")
+
+    # Angular Axis
+    if all(isinstance(t, str) for t in theta):
+        if angular_categoryarray is not None:
+            symbolic_theta = list(angular_categoryarray)
+        else:
+            symbolic_theta = list(dict.fromkeys(theta))
+
+        if angular_categoryorder is not None:
+            if angular_categoryorder == "category ascending":
```

Comment:
@jsaussereau do you have example of plotly figure where this attribute is used ? I did not manage to find such one